src/app/interfaces/dataset.interface.ts

```diff
@@ -158,3 +159,17 @@ export interface DatasetUpdate {
   create_ancestors?: boolean;
   user_properties_update?: { key: string; value: string; remove?: boolean }[];
 }
+
+export interface DatasetDetails extends Omit<Dataset, 'children'> {
```

Comment:
Are other properties from Dataset really available here. I think it may be better to create a completely new interface and remove `DatasetInTree`